client/tests/unit/landing.spec.ts

```diff
@@ -63,7 +66,10 @@ describe('landing.vue', () => {
 describe('landing.vue', () => {
   let wrapper: any
 
-  beforeEach(() => {
+  beforeAll(() => {
+    newReqModule.mutateRequestAction('NEW')
+    newReqModule.mutateLocation('BC')
+    newReqModule.mutateEntityType('CR')
     newReqModule.mutateDisplayedComponent('AnalyzeResults')
 
     wrapper = mount(Landing, {
```

Comment:
Again, no need to change anything right now, but for next time keep in mind that usually:
- beforeEach() will mount the component under test
- afterEach() will destroy the wrapper (ie, unmount and clean up memory)